src/scripts/Build-Toolkit.ps1

```diff
@@ -85,11 +85,17 @@ Get-ChildItem "$PSScriptRoot/../workbooks/$($Template -replace '-workbook$','')*
 | ForEach-Object { Build-QuickstartTemplate $_ }
 
 # Package templates
-Get-ChildItem "$PSScriptRoot/../templates/$Template*" -Directory -ErrorAction SilentlyContinue `
+Get-ChildItem -Path "$PSScriptRoot/../templates/*", "$PSScriptRoot/../optimization-engine*" -Directory -ErrorAction SilentlyContinue `
```

Comment:
@helderpinto Including you since I moved this code around. As far as I could tell, the only difference between AOE and hubs was the bicep build step, so I just filtered that out when the file doesn't exist. We can get more specific later, if needed.

Reply:
Looks good. Much simpler and cleaner now :-) I tested the change and AOE was correctly built.